backend/src/controllers/picker_options.js

```diff
@@ -0,0 +1,98 @@
+const log = require("../facilities/logging.js")(module.filename);
+const Model = require("../models/picker_options.js");
+const what = { single: "picker_options", plural: "picker_optionss" };
+
+/**
+ * Checks to see if a user access a route based on the allowedRole.
+ *
+ * @param   {FastifyRequest} request    The request object, which should have the user capability via the fastify-roles plugin.
+ * @param   {string}         capability Is the name of the role that is required to access the route.
+ * @returns {boolean}
+ */
+const userCan = (request, capability) => {
+  const userCapabilities = request?.user?.capabilities || [];
+  return userCapabilities.includes(capability);
+};
+
+/**
+ * This is a helper function that returns 401 with generic message if user is not allowed to access route.
+ *
+ * @param   {FastifyReply} reply The reply object, in order to set the status code.
+ * @returns {object}
+ */
+const notAllowed = (reply) => {
+  reply.code(401);
+  return { message: `You don't have the correct permission` };
+};
+
+/**
+ * For roles that might require only if mine, however this still needs to be implemented.
+ *
+ * @param   {FastifyRequest} request The request object
+ * @todo  Add functionality to call db to see if the owner is the current user.
+ * @returns {boolean}
+ */
+const checkMine = (request) => {
```

Comment:
Picker options shouldn't have owners, right? It's not like another DB object that was created by a user, these options should be shipped with the app, shouldn't they?

Reply:
Probably not, this was generated with the command we built so it's just MVP.